### PR TITLE
fix: Normalize Windows line breaks before parsing configuration files

### PR DESCRIPTION
**Pull request by erezrokah**
#### Summary

Fixes https://github.com/cloudquery/plugin-sdk/issues/351

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: plugins/source_docs_test.go [hunk 1]
```diff
@@ -1,3 +1,5 @@
+//go:build !windows
```
Comment (erezrokah) [inline plugins/source_docs_test.go:1]: We don't want cross platform snapshot tests 🐰 🕳️

File: codegen/template_test.go [hunk 2]
```diff
@@ -1,3 +1,5 @@
+//go:build !windows
```
Comment (erezrokah) [inline codegen/template_test.go:1]: We don't want cross platform snapshot tests 🐰 🕳️

File: clients/destination_test.go [hunk 3]
```diff
@@ -1,3 +1,5 @@
+//go:build !windows
```
Comment (erezrokah) [inline clients/destination_test.go:1]: See issue https://github.com/cloudquery/plugin-sdk/issues/354

File: clients/source_test.go [hunk 4]
```diff
@@ -1,3 +1,5 @@
+//go:build !windows
```
Comment (erezrokah) [inline clients/source_test.go:1]: Same issue https://github.com/cloudquery/plugin-sdk/issues/354

**Comment by erezrokah** [comment]
Before we merge this we'll need to update the branch protection rules

**Review by hermanschaaf — APPROVED** [review]
👍

**Review by disq — APPROVED** [review]

**Review by yevgenypats — APPROVED** [review]

**Pull request merged**